### ISSUE TITLE
Gap between the performance of training and validation

### ISSUE THREAD

**Issue by wmmxk** [lixiaojieff/HGTAN#6]
Thanks for publishing your code.

I run train_Transformer.py for 600 epochs. I observed the training loss is far less than the validation loss, and the accuracy on the validation loss is 39%. Below is what I copied from lstm+trans+HGAT3_5_valid1.train.log  and lstm+trans+HGAT3_5_valid1.valid.log 

Train:
epoch, loss,  accuracy,percision,recall,f1_score
 598, 0.99904,55.527,  48.405,  46.126,  48.076
 599, 0.99891,55.464,  46.769,  45.647,  46.475

Validation:
epoch, loss,  accuracy,percision,recall,f1_score
 598, 1.11303,37.502,  41.162,  38.029,  36.878
 599, 1.10299,39.434,  43.022,  41.418,  43.035

So I was wondering whether you got a similar log file when running the code.
